src/scripts/main.js

```diff
@@ -354,7 +354,35 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach((person) => {
+  const lineContainer = document.createElement('tr');
+  const lineName = document.createElement('th');
+  const lineGender = document.createElement('th');
+  const lineBorn = document.createElement('th');
+  const lineDied = document.createElement('th');
+  const lineAge = document.createElement('th');
+  const lineCentery = document.createElement('th');
+  const lines = [
+    lineName,
+    lineGender,
+    lineBorn,
+    lineDied,
+    lineAge,
+    lineCentery,
+  ];
+
+  lineName.innerText = person.name;
+  lineGender.innerText = person.sex === 'm' ? 'Male' : 'Female';
+  lineBorn.innerText = person.born;
+  lineDied.innerText = person.died;
+  lineAge.innerText = person.died - person.born;
+  lineCentery.innerText = Math.ceil(person.died / 100);
```

Comment:
The calculation for the century seems to be off by one for years that are exactly on the century mark (e.g., someone who died in 1900 should be in the 19th century, not the 20th). You should adjust the calculation to account for this edge case.